aspnetcore/toc.yml

```diff
@@ -773,6 +773,8 @@
           uid: grpc/deadlines-cancellation
         - name: Transient fault handling
           uid: grpc/retries
+        - name: Client-side load balancing
```

Comment:
Although there is a doc build warning here that the `grpc/loadbalancing" cross reference cannot be found, I suspect the real issue here is that we had validation errors with the metadata on the loadbalancing.md doc and as a result, the link cannot be resolved yet.  The entry in toc.yml looks fine to me.

Reply:
All fixed now. No validation issues.